### PR TITLE
Render system removeable fix (Merge after fix-entity-removal)

### PR DESCRIPTION
**Pull request by skumbananaaa**
Fixes Removal of Entities in the RenderSystem

### CODE REVIEW

File: Assets/Shaders/Defines.glsl [hunk 1]
```diff
@@ -80,12 +80,18 @@ struct SAreaLight
     uvec2   Padding;
 };
 
+struct SPointLight
+{
+    vec4    ColorIntensity;
+    vec3    Position;
+    uint    padding;
```
Comment (Vonsen) [inline Assets/Shaders/Defines.glsl]: Capital letter

File: LambdaEngine/Include/Rendering/Core/API/GraphicsTypes.h [hunk 2]
```diff
@@ -27,7 +27,7 @@ namespace LambdaEngine
 	constexpr const uint32 BACK_BUFFER_COUNT			= 3;
 
 	// Determines if a resource should be allocated by a deviceallocator or via a seperate allocation
-	constexpr const uint32 LARGE_TEXTURE_ALLOCATION_SIZE				= MEGA_BYTE(64);
+	constexpr const uint32 LARGE_TEXTURE_ALLOCATION_SIZE				= MEGA_BYTE(128);
```
Comment (norda97) [inline LambdaEngine/Include/Rendering/Core/API/GraphicsTypes.h:30]: Texture allocation can be set to 64 MB

File: LambdaEngine/Source/Rendering/RenderGraph.cpp [hunk 3]
```diff
@@ -613,7 +613,7 @@ namespace LambdaEngine
 						}
 						else
 						{
-							uint32 actualSubResourceCount = pResource->Texture.IsOfArrayType ? 1.0f : pResource->SubResourceCount;
+							uint32 actualSubResourceCount = pResource->Texture.IsOfArrayType ? 1 : pResource->SubResourceCount;
```
Comment (norda97) [inline LambdaEngine/Source/Rendering/RenderGraph.cpp]:
```suggestion
							uint32 actualSubResourceCount = pResource->Texture.IsOfArrayType ? 1U : pResource->SubResourceCount;
```

File: LambdaEngine/Include/Rendering/Core/API/GraphicsTypes.h (hunk 2 above)
Comment (norda97) [inline LambdaEngine/Include/Rendering/Core/API/GraphicsTypes.h]:
```suggestion
	constexpr const uint32 LARGE_TEXTURE_ALLOCATION_SIZE				= MEGA_BYTE(64);
```

**Review by norda97 — APPROVED** [review]
Good work, commit suggested changes.

**Comment by norda97** [comment]
I don't think this branched pulled from master before being requested as pull-request

**Pull request merged**